rst2pdf/opt_imports.py

```diff
@@ -69,42 +69,5 @@
 except ImportError:
     mathtext = None
 
-class LazyImports(object):
-    ''' Only import some things if we need them.
-    '''
-
-    def __getattr__(self, name):
-        if name.startswith('_load_'):
-            raise AttributeError
-        value = None
-        if not os.environ.get('DISABLE_' + name.upper()):
-            func = getattr(self, '_load_' + name)
-            try:
-                value = func()
-            except ImportError:
-                pass
-        # Cache the result once we have it
-        setattr(self, name, value)
-        return value
-
-    def _load_pdfinfo(self):
-        try:
-            from pyPdf import pdf
-        except ImportError:
-            import pdfrw as pdf
-        return pdf
-
-    def _load_PILImage(self):
-        try:
-            from PIL import Image as PILImage
-        except ImportError:
-            import Image as PILImage
-        return PILImage
-
-    def _load_svg2rlg(self):
-        if six.PY2:
-            import svg2rlg
-            return svg2rlg
-        return None
-
-LazyImports = LazyImports()
+import pdfrw as pdfinfo
+from PIL import Image as PILImage
```

Comment:
What is the benefit of importing these two within opt_imports compared to importing directly in `image.py`? 

Or, to put it another way, should there be a `try` around these imports?

Reply:
Just wanted to make the change minimal.

They don't need a try/except because we are really requiring them now :-)

Reply:
Got it. My gut feeling is that they shouldn't be in opt_imports as they aren't optional. Your call though as you know this better than I do.